omnibus/config/software/gmp.rb

```diff
@@ -33,7 +33,8 @@
 
 build do
   env = with_standard_compiler_flags(with_embedded_path)
-  env["CFLAGS"] << " -fPIC"
+  # With `--enable-flat`, CPU feature detection is done at runtime, improving compatibility
+  env["CFLAGS"] << " -fPIC --enable-fat"
 
   configure "--disable-static", env: env
```

Comment:
```suggestion
  configure "--disable-static --enable-fat", env: env
```
this should work better

Reply:
Indeed, good catch.